markdown/blog/community-spotlight-ashmit.md

```diff
@@ -0,0 +1,136 @@
+  - Spotlight
+  - Community
+cover: /img/posts/marketing-images/ashmit-story.webp
+authors:
+  - name: Atinuke Oluwabamikemi Kayode
+    photo: /img/avatars/bami.webp
+    link: https://www.linkedin.com/in/atinuke-oluwabamikemi-kayode-5b838b1b7/
+    byline: AsyncAPI Community Marketing Specialist
+excerpt: 'Being patient is important as a maintainer because sometimes the contributors don't have it, so you have to keep yours. Says, Ashmit. A third-year Bachelor of Technology student in Computer Science from India.'
+---
+
+
+Open-source contributions have long been a gateway for developers to refine their skills, build meaningful projects, and connect with a global community. For many, the journey starts with small contributions, but for some, it evolves into a leadership role that impacts the project’s future. One such inspiring story is that of [Ashmit Jagtap](https://www.linkedin.com/in/ashmit-jagtap/), whose path from a Google Summer of Code (GSoC) mentee to a maintainer within AsyncAPI showcases dedication, learning, and a deep commitment to open-source development. In this blog post, we explore his journey, the challenges he faced, and the lessons he learned along the way.
+
+## The GSoC Experience: A Transformative Phase
+
+Ashmit’s open-source journey began with his participation in GSoC, an esteemed program that connects aspiring developers with organizations working on impactful projects. His decision to contribute to the AsyncAPI Initiative was fueled by his interest in API documentation and event-driven architectures. As a mentee, he worked on refining the AsyncAPI Conference website and was mentored by experienced contributors who provided guidance, code reviews, and architectural insights.
+
+Reflecting on his GSoC experience, Ashmit emphasizes the steep learning curve and the importance of proactive engagement. He recalls, “Initially, I was overwhelmed by the scale of the project, but the community was incredibly supportive. I learned how to navigate large codebases, improve my problem-solving skills, and contribute in a structured manner.” This phase allowed him to grasp not only the technical aspects of the project but also the collaborative spirit of open-source communities.
+
+We then asked Ashmit some questions about his GSoC experience with the AsyncAPI Initiative. Below is an outline of our conversation.
+
+## Getting Started with Google Summer of Code and AsyncAPI
+
+**Bami: What inspired you to apply for Google Summer of Code with AsyncAPI?**
+
+***Ashmit:** Google Summer of Code had been a goal of mine since my first year of college when I first learned about it. However, I knew it was a competitive program, so I focused on building my skills. By my second year, I was ready to start contributing and began working towards it in January. Initially, I aimed to contribute to Postman, which serves as the umbrella organization for AsyncAPI and JSON Schema. My contributions to both projects led to my selection in AsyncAPI.*
```

Comment:
```suggestion
***Ashmit:** Google Summer of Code had been a goal of mine since my first year of college when I first learned about it. However, I knew it was a competitive program, so I focused on building my skills. I was ready to contribute by my second year and began working towards it in January. Initially, I aimed to contribute to Postman, but my contributions to both projects led to my selection for AsyncAPI.*
```